formatter.go

```diff
@@ -0,0 +1,52 @@
+package logger
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	fieldKeyTime        = "@time"
+	fieldKeyLogTime     = "logTime"
+	fieldKeyServiceName = "service_name"
+)
+
+type ftJSONFormatter struct {
+	serviceName string
+}
+
+func newFTJSONFormatter(serviceName string) logrus.Formatter {
+	return &ftJSONFormatter{
+		serviceName: serviceName,
+	}
+}
+
+func (f *ftJSONFormatter) Format(entry *logrus.Entry) ([]byte, error) {
+	data := make(logrus.Fields, len(entry.Data)+4)
+	for k, v := range entry.Data {
+		switch v := v.(type) {
+		case error:
+			// Otherwise errors are ignored by `encoding/json`
+			data[k] = v.Error()
+		default:
+			data[k] = v
+		}
+	}
+
+	if _, found := data[fieldKeyTime]; found {
+		data[fieldKeyLogTime] = entry.Time.Format(timestampFormat)
```

Comment:
I think we can remove this case from here. I would only leave the check for the @time field. If it's in the entries, let's just keep it, otherwise let's overwrite it with the actual formatted time. The logTime - or any other time related field can be added by the service that decides to do that.